lib/task/tasks.rb

```diff
@@ -0,0 +1 @@
+require_relative 'tasks/composite_task.rb'
```

Comment:
`require 'task/tasks/composite_task.rb'` is actually less characters :trollface: